data/421/172/813/421172813.geojson

```diff
@@ -129,6 +129,9 @@
         "C\u00f3rdoba"
     ],
     "name:fra_x_preferred":[
+        "Cordoue"
```

Comment:
Should `name:eng_x_preferred` and `wof:name` also be updated?

Reply:
I'm actually unable to verify English names in much of this localization work.. so in most PRs, I'll be skipping English/`wof:name` updates..